docs/deploy/images.md

```diff
@@ -0,0 +1,21 @@
+# Container Images
+
+Images for the various KubeRay components are published at the following locations:
+
+1. [Quay.io](https://quay.io/organization/kuberay)
+2. [Dockerhub](https://hub.docker.com/u/kuberay)
+
+We recommend using Quay as the primary source for images as there are are no image-pull restrictions on Quay.io.
```

Comment:
Would you mind adding more details about "image-pull restrictions"?

Reply:
Updated